frontend/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
maybe not commiting those frontend files who would override the work of Sangeetha